lighthouse-cli/printer.ts

```diff
@@ -118,37 +125,38 @@ function createOutput(results, outputMode) {
   let output = `\n\n${bold}Lighthouse (${version}) results:${reset} ${results.url}\n\n`;
 
   results.aggregations.forEach(aggregation => {
+
     output += `▫ ${bold}${aggregation.name}${reset}\n\n`;
 
     aggregation.score.forEach(item => {
       const score = (item.overall * 100).toFixed(0);
 
       if (item.name) {
-        output += `${bold}${item.name}${reset}: ${item.scored ? formatScore(score, '%') : ''}\n`;
+        output += `${bold}${item.name}${reset}: ${item.scored ? formatAggregationResultItem(score, '%') : ''}\n`;
       }
 
       item.subItems.forEach(subitem => {
-        // Get audit object from inside of results.audits under name subitem.
-        // Coming soon events are not located inside of results.audits.
-        subitem = results.audits[subitem] || subitem;
+        let subscore: AuditResult;
```

Comment:
1. `subScore` doesn't make sense as a name for AuditResult objects.
2. you nuked `if (subitem.comingSoon) return;` and don't handle it. you can address your above conflict by filtering out the comingSoon guys this way, eh?